src/common.rkt

```diff
@@ -185,7 +185,7 @@
   (cond 
     [(not r) ""]
     [else
-      (define val (~r (/ (round (* r 10)) 10) #:precision 2))
+      (define val (~r (/ (round (* r 10)) 10) #:precision '(= 2)))
```

Comment:
Take a look at the actual thing being printed: it's doing an explicit round to 1 decimal place. So then printing to 2 decimal places makes no sense.

Reply:
Also, as far as I can tell, we only use `format-cost` in three places:

1. In the large number on `make-graph`, where it represents a cost, which is always an integer
2. In the "alternatives" portion of `make-graph`, where it represents a cost, which is always an integer
3. In `timeline` in the `regimes` phase, where it represents a *change in accuracy*

I say we get rid of this method entirely. Cases 1 and 2 don't need special printing code, and we can inline the helper into case 3.

Reply:
Makes sense. Will revert the format-cost change.